Doc/library/math.rst

```diff
@@ -10,9 +10,6 @@
 
 --------------
 
-This module provides access to the mathematical functions defined by the C
-standard.
-
 These functions cannot be used with complex numbers; use the functions of the
```

Comment:
Now this sentence seems weird. What "functions"? I think we should say that provides access to mathematical functions defined by the C standard **and others**. Or we can use the same sentence as for `math.h` on https://cplusplus.com/reference/cmath/:

>  Header <cmath> declares a set of functions to compute common mathematical operations and transformations:

Reply:
The "mathematical functions" in `math` that were just written in big letters :)

<img width="832" alt="image" src="https://github.com/user-attachments/assets/1d1aff3f-6ff7-430b-a2ef-520eb91cda92" />

https://cpython-previews--134621.org.readthedocs.build/en/134621/library/math.html

Do you think it's not clear?

Reply:
Like this?

```suggestion
The functions in this module cannot be used with complex numbers; use the functions of the
```

Reply:
I think it's better. However, I feel like it's a bit abrupt as an introduction because we directly say "this module cannot do that". While it could be overpedantic, I would have a sentence like that:

```rst
This module provides access to common mathematical functions and constants. These functions cannot however be used with complex numbers; use the functions of the [...]
```

Reply:
@skirpichev What do you suggest?

Reply:
Added in 1043f269d5da492d7af798912aeb2b9f8551178e.